metron-platform/metron-enrichment/src/main/java/org/apache/metron/enrichment/bolt/GenericEnrichmentBolt.java

```diff
@@ -179,6 +180,7 @@ public void declareOutputFields(OutputFieldsDeclarer declarer) {
   @SuppressWarnings("unchecked")
   @Override
   public void execute(Tuple tuple) {
+    long texecute1 = System.currentTimeMillis();
```

Comment:
I don't know if we actually care given that it's logging, but iirc `currentTimeMillis()` returns can differ based on ntp syncs and so on (end time can be before start time, for example).  `nanotime()` is generally more reliable for this, but it can also be more expensive (on the order of millis).

Even with `currentTimeMillis()`, I think the actual call can still end up being slow (milli+ depending on the system).  It might be worthwhile to wrap the timing calls in a debug guard given that these run in the `execute()`, but I'm not sure if it'll make a difference in practice.